### PR TITLE
Add COSMIC Protocols

### PR DESCRIPTION
**Pull request by bbb651**
Adds the [cosmic protocols](https://github.com/pop-os/cosmic-protocols) (still missing the compositor support info). I'm not sure about the order it should appear in, I placed it after everything else but weston and external.
I'm also not sure about the badge colors, I couldn't get a light background and dark text to work as it didn't look like the cosmic orange accent color and was too close to in appearance to weston, so I used the accent color as the background.
I ran into [this](https://github.com/facebook/create-react-app/issues/12409) while debugging which made it really annoying to test, did you also ran into this issue?

### CODE REVIEW

File: src/components/content/ProtocolBadge.tsx [hunk 1]
```diff
@@ -36,6 +36,11 @@ const kdeProtocolsUnstableBadgeTheme: BadgeTheme = {
     backgroundColor: 'bg-purple-100',
 }
 
+const cosmicProtocolsUnstableBadgeTheme: BadgeTheme = {
+    textColor: 'text-amber-100',
+    backgroundColor: 'bg-orange-400',
```
Comment (vially) [inline src/components/content/ProtocolBadge.tsx:41]: I find the contrast between these two colors a bit hard-to-read:

20240825_15h20m44s_grim

I see what you mean about being difficult to find some colors that match the COSMIC logo though. I've tried a few combinations and the best options I could find are either `amber` or `yellow` (for both text and background).

### yellow

```yaml
textColor: 'text-yellow-100'
backgroundColor: 'bg-yellow-800'
```

20240825_15h09m51s_grim

### amber

```yaml
textColor: 'text-amber-100'
backgroundColor: 'bg-amber-800'
```

20240825_15h10m35s_grim

Neither of them is great, but for these two options the contrast seems to be slightly better.  I don't have strong opinions on it though so I'm fine with either one of these 3 options in the end.

**Review by vially — CHANGES_REQUESTED** [review]
Looks great, thanks a lot for the contribution. The `eslint-disable-next-line` comment is required to fix the CI pipeline. Once that's fixed i'll approve and merge it.

**Comment by bbb651** [comment]
You are right about the contrast, I forgot to check it and it's really low:
image
I also tried pure white text and it's not much better:
image
Your suggestions:
- Normal:
image
image

- With white text instead:
image
image

I don't have a strong opinion about it either and my display has terrible color accuracy. Maybe it's better to just go for a light background if the color can't be close enough to the right one. I'll ask in the PopOS Mattermost if they have a preference.

(The app I used is [Contrast](https://flathub.org/apps/org.gnome.design.Contrast))

**Comment by bbb651** [comment]
They did not answer so I assume it's not that important, I was probably overthinking it. I switched it to a light background to be more consistent with the other badges, and it still looks orange and is distinguishable enough from weston (at least for me), and has good enough contrast. 
image

**Review by vially — APPROVED** [review]
Looks great, thanks for your contribution! :tada:

**Comment by bbb651** [comment]
Thanks! You can close #67. Also I just made one for compositor support data #78

**Pull request merged**